common/djangoapps/contentserver/middleware.py

```diff
@@ -55,16 +56,32 @@ def process_request(self, request):
             except (ItemNotFoundError, NotFoundError):
                 return HttpResponseNotFound()
 
+            # Set the basics for this request.
+            newrelic.agent.add_custom_parameter('contentserver.course_key', loc.course_key)
```

Comment:
Please call this `course_id` instead. Both for consistency, and so that we can do a more global "facet by course_id" to see what courses are eating up resources across different views.

Similarly, please log the org.